app/src/main/res/layout/fragment_create_personal_account.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingStart="@dimen/screen_padding"
+    android:paddingEnd="@dimen/screen_padding">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/createPersonalAccountTopGuideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_percent="0.228" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/createPersonalAccountTitleTextView"
+        style="@style/TextView.Authentication.Header"
```

Comment:
Is this formatted? style attributes normally get auto-formatted to the top by default.

Reply:
nope, this is the default format. it goes roughly: id, style, layout_width/height, others...

Reply:
That's my bad, was looking at a file without android:id, ignore me.